go.mod

```diff
@@ -3,7 +3,8 @@ module github.com/webrpc/webrpc
 go 1.16
 
 require (
-	github.com/pkg/errors v0.9.1
+	github.com/golang-cz/textcase v1.2.0
+	github.com/pkg/errors v0.8.1
```

Comment:
btw -- can you please rebase from master, and also avoid using pkg/errors in the code and use eithers just `fmt.Errorf("etc")`, `fmt.Errorf("etc: %w", err)` or `errors.New("..")`